src/nvim/eval.c

```diff
@@ -22041,16 +22041,18 @@ void func_dump_profile(FILE *fd)
         } else {
           fprintf(fd, "FUNCTION  %s()\n", fp->uf_name);
         }
-        bool should_free;
-        const LastSet last_set = (LastSet){
-          .script_ctx = fp->uf_script_ctx,
-          .channel_id = 0,
-        };
-        char_u *p = get_scriptname(last_set, &should_free);
-        fprintf(fd, "    Defined: %s line %" PRIdLINENR "\n",
-                p, fp->uf_script_ctx.sc_lnum);
-        if (should_free) {
-          xfree(p);
+        if (fp->uf_script_ctx.sc_sid != 0) {
```

Comment:
[vim-patch:8.1.1946](https://github.com/vim/vim/commit/163588005da3a240e49416093d0d0251951d60a1)

Reply:
Changes to `test_profile.vim` in [vim-patch:8.1.1946](https://github.com/vim/vim/commit/163588005da3a240e49416093d0d0251951d60a1) cannot be imported due to lack of patches. Isn't it necessary to reflect changes to `test_profile.vim` because it is a test that is not run?

Reply:
That test is N/A so you can omit the test. Comment out the line, `source screendump.vim`.

Reply:
We could have an empty `screendump.vim` so that such lines don't need to be commented-out.